analyzer/consensus/consensus.go

```diff
@@ -138,7 +138,7 @@ func (m *Main) Start() {
 	}
 
 	backoff, err := util.NewBackoff(
-		100*time.Millisecond,
+		0,
```

Comment:
would we ever want nonzero initial "timeout"? seems like it was used wrong later on, we should only `backoff.Wait()` on errors, rather than in every iteration

Reply:
oh although if we did that we wouldn't have this gradual speedup when an error stops happening

Reply:
Refactored the struct a bit. This `100*time.Millisecond` now defines the "minimum timeout step".